src/main/java/com/digitax/controller/AuthController.java

```diff
@@ -571,8 +671,51 @@ else if (userRepository.existsByUsername(loginRequest.getUsername())) {
 		    return new ResponseEntity<>(ApiRes.success(e.getMessage(), statusObj), HttpStatus.OK);	
 			 }	      
           }
-    
-    
+    /**#
+     * 
+     * @param request
+     * @param bindingResult
+     * @return
+     */
+
```

Comment:
@Jayanta-Redapple [Minor] You need to remove the space between the javadoc and the code in order for the connection to be made. Right now as it is, this is a "floating comment". Means the IDE cannot connect that documentation to the code.

Reply:
Sure @leonardtatum , I am working on it, If it is not necessary I will remove it